restli-client/src/test/java/com/linkedin/restli/client/RestClientTest.java

```diff
@@ -176,6 +176,7 @@ private TimeoutOption(Long l, TimeUnit timeUnit)
     LICOR_TEXT(ContentType.LICOR_TEXT),
     LICOR_BINARY(ContentType.LICOR_BINARY),
     PROTOBUF(ContentType.PROTOBUF),
+    LI_PROTOBUF(ContentType.LI_PROTOBUF),
```

Comment:
comment here a new content type due to backward incompatible for existing ProtoReader.

Reply:
Rather than add a comment here, I added it in content-type.

Reply:
Then adding a comment there saying why we have two: PROTOBUF_DATA_CODEC and LI_PROTOBUF_DATA_CODEC. and difference btw the two.